.changeset/dry-ligers-chew.md

```diff
@@ -0,0 +1,6 @@
+---
+"@ledgerhq/react-ui": minor
+"@ledgerhq/icons-ui": patch
```

Comment:
shouldn't it be a minor since it's a removal (so it's breaking)?

Reply:
or actually major 🤔

https://semver.org/#summary
```
MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backwards compatible manner
PATCH version when you make backwards compatible bug fixes
```

Reply:
Not applicable as UILib is still in alpha cycles. Putting major here would make it pass from a v0 to a v1. I did put minor on the ReactUI which is the endpoint where users would import the component, IconsUI is just a source for ReactUI.